CMakeLists.txt

```diff
@@ -13,18 +13,11 @@
 # limitations under the License.
 
 cmake_minimum_required(VERSION 3.17.2)
```

Comment:
See docs for [cmake_minimum_required](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html?highlight=cmake_minimum)

> Sets the minimum required version of cmake for a project. Also updates the policy settings as explained below.